src/h3/_cy/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+find_package(Cython MODULE REQUIRED)
+find_package(PythonExtensions MODULE REQUIRED)
+
+include_directories(${CMAKE_CURRENT_SOURCE_DIR})
+
+macro(add_cython_file filename)
+    add_cython_target(${filename} C PY3)
+    add_library(${filename} MODULE ${filename} ${LIB_SOURCE_FILES} ${CONFIGURED_API_HEADER})
+    python_extension_module(${filename})
+    set_property(TARGET ${filename} PROPERTY C_STANDARD 99)
+    target_compile_definitions(${filename} INTERFACE $<TARGET_PROPERTY:h3,COMPILE_DEFINITIONS>)
+    target_include_directories(${filename} INTERFACE $<TARGET_PROPERTY:h3,INCLUDE_DIRECTORIES>)
+    target_link_libraries(${filename} h3)
+    install(TARGETS ${filename} LIBRARY DESTINATION src/h3/_cy)
+endmacro()
+
+# GLOB pattern is reccomended against
```

Comment:
nit typo reccommended => recommended

Reply:
that's not a nit; that's just plain wrong!

fixed!